index.js

```diff
@@ -30,8 +53,15 @@ const action = async context => {
 	});
 
 	const response = await upload.promise();
+	let uploadURL = response.Location;
+
+	const baseURL = context.config.get('baseURL');
+	console.log('baseURL' + baseURL);
```

Comment:
Remove the log here.

Reply:
Removed here: https://github.com/SamVerschueren/kap-s3/pull/3/commits/cc67b6844187732204eae4b4bbfffcced019be0a